worlds/hades/Rules.py

```diff
@@ -0,0 +1,420 @@
+from BaseClasses import MultiWorld, Item, ItemClassification
+from .Items import item_table_pacts, item_table_weapons, item_table_keepsake, items_table_fates_completion, HadesItem
+from ..AutoWorld import LogicMixin
+from ..generic.Rules import set_rule, add_rule, add_item_rule
```

Comment:
```suggestion
from worlds.AutoWorld import LogicMixin
from worlds.generic.Rules import set_rule, add_rule, add_item_rule
```
Need to use absolute imports when importing from outside of your apworld